public/scripts/reducers.js

```diff
@@ -66,6 +66,23 @@ let myFormReducer = (theirFormReducer) => {
       case Action.SELECT_PROJECT:
         const {id} = action.payload;
         state.projectSelector.project.value = id ? id.toString() : "";
+      case Action.CHECK_UNCHECK_FEATURES:
+        let field_names = Object.keys(state.featurize).filter(
+          fn => fn.startsWith(action.payload))
+        let formState = Object.assign({}, state.featurize)
+        let allAreChecked = (() => {
+          for (var idx in field_names) {
```

Comment:
Wouldn't you have to create an `array` first to be able to use find? Seems like it would actually be less clean to do that